lib/matplotlib/ticker.py

```diff
@@ -1982,15 +1976,6 @@ def decade_up(x, base=10):
     return base ** lx
 
 
-def nearest_long(x):
```

Comment:
`round` does banker's rounding in Python 3; `nearest_long` does not.

Reply:
...yep, that's not `round`.  It's `int`.  /facepalm.

Reply:
Sorry for being unclear; it's not `int` either, it's whatever the other kind of rounding is.

Reply:
Yep, round-half-to-zero.  Fixed by inlining the logic in the two places it was needed, and just using `round` when we know the value is very close to an integer.

Reply:
Also, public api, yada yada.